assets/kube-burner-report-ocp-wrapper/queries.libsonnet

```diff
@@ -0,0 +1,1589 @@
+        + elasticsearch.withBucketAggs([
+          elasticsearch.bucketAggs.Terms.withField("labels.mode.keyword")
+          + elasticsearch.bucketAggs.Terms.withId("3")
+          + elasticsearch.bucketAggs.Terms.withType('terms')
+          + elasticsearch.bucketAggs.Terms.settings.withOrder('desc')
+          + elasticsearch.bucketAggs.Terms.settings.withOrderBy('1')
+          + elasticsearch.bucketAggs.Terms.settings.withMinDocCount("1")
+          + elasticsearch.bucketAggs.Terms.settings.withSize("10"),
+          elasticsearch.bucketAggs.DateHistogram.withField("timestamp")
+          + elasticsearch.bucketAggs.DateHistogram.withId("2")
+          + elasticsearch.bucketAggs.DateHistogram.withType('date_histogram')
+          + elasticsearch.bucketAggs.DateHistogram.settings.withInterval('auto')
+          + elasticsearch.bucketAggs.DateHistogram.settings.withMinDocCount('1')
+          + elasticsearch.bucketAggs.DateHistogram.settings.withTrimEdges(0),
+        ])
+        + elasticsearch.withMetrics([
+         elasticsearch.metrics.MetricAggregationWithSettings.Average.withField("value")
+         + elasticsearch.metrics.MetricAggregationWithSettings.Average.withId("1")
+         + elasticsearch.metrics.MetricAggregationWithSettings.Average.withSettings({
+            "script": {
+              "inline": "_value*100"
+            }
+          })
+         + elasticsearch.metrics.MetricAggregationWithSettings.Average.withType('avg'),
+        ])
+        + elasticsearch.withQuery("uuid.keyword: $uuid AND metricName.keyword: \"nodeCPU-Masters\" AND labels.instance.keyword: $master")
+        + elasticsearch.withTimeField('timestamp')
+  },
+  masterMemory: {
+    base(alias, query):
+        elasticsearch.withAlias(alias)
+        + elasticsearch.withBucketAggs([
+          elasticsearch.bucketAggs.DateHistogram.withField("timestamp")
+          + elasticsearch.bucketAggs.DateHistogram.withId("2")
+          + elasticsearch.bucketAggs.DateHistogram.withType('date_histogram')
+          + elasticsearch.bucketAggs.DateHistogram.settings.withInterval('30s')
+          + elasticsearch.bucketAggs.DateHistogram.settings.withMinDocCount('1')
+          + elasticsearch.bucketAggs.DateHistogram.settings.withTrimEdges(0),
+        ])
+        + elasticsearch.withMetrics([
+         elasticsearch.metrics.MetricAggregationWithSettings.Average.withField("value")
+         + elasticsearch.metrics.MetricAggregationWithSettings.Average.withId("1")
+         + elasticsearch.metrics.MetricAggregationWithSettings.Average.withType('avg'),
+        ])
+        + elasticsearch.withQuery(query)
+        + elasticsearch.withTimeField('timestamp'),
+    queries(): [
+      self.base("Available", "uuid.keyword: $uuid AND metricName.keyword: \"nodeMemoryAvailable-Masters\" AND labels.instance.keyword: $master"),
+      self.base("Total", "uuid.keyword: $uuid AND metricName.keyword: \"nodeMemoryTotal-Masters\" AND labels.instance.keyword: $master"),
```

Comment:
Just removed these two today, we're not indexing it anymore

Reply:
Done.